src/nss-systemd/nss-systemd.c

```diff
@@ -329,7 +334,7 @@ enum nss_status _nss_systemd_setgrent(int stayopen) {
         getgrent_data.iterator = userdb_iterator_free(getgrent_data.iterator);
         getpwent_data.by_membership = false;
 
-        r = groupdb_all(nss_glue_userdb_flags(), &getgrent_data.iterator);
+        r = groupdb_all(nss_glue_userdb_flags() | USERDB_DONT_SYNTHESIZE, &getgrent_data.iterator);
```

Comment:
Maybe a comment here to look at the comment block in `_nss_systemd_setpwent`?

Reply:
sure why not